### ISSUE TITLE
Android textinput bug

### ISSUE THREAD

**Issue by arifsuheyl** [gbumps/react-native-screenguard#62]
Hi,
In IOS this works like a charm, but in android there is a problem. Although it prevents recording, when I click on textinput, keyboard is not opening. I check that it is focused but keyboard is not seen. How can I fix this problem?

**Comment by gbumps** [comment]
Hi @arifsuheyl, I have noted on Readme how to temporarily solved this problem 
on Android. 

```

Please remember that text input will be temporary disabled until calling unregister on Android except registerWithoutEffect. Workaround: You can use your own custom-made keyboard if included in your project, or find some react native libs which provide custom keyboard ui like wix/react-native-ui-libs could solve this.


```

**Comment by gbumps** [comment]
Hi @arifsuheyl, does it solve your problem yet, if not, you can reopening this anytime you want.

**Comment by Kirillsocivka** [comment]
> Hi [@arifsuheyl](https://github.com/arifsuheyl), I have noted on Readme how to temporarily solved this problem on Android.
> 
> ```
> 
> Please remember that text input will be temporary disabled until calling unregister on Android except registerWithoutEffect. Workaround: You can use your own custom-made keyboard if included in your project, or find some react native libs which provide custom keyboard ui like wix/react-native-ui-libs could solve this.
> ```

I'm using the following packages:

react-native-gifted-chat: 2.8.1
react-native": "0.79.1"
react-native-screenguard: 1.0.8-beta

I'm facing a problem: I have chat functionality in my app, and I need to restrict screenshot access for specific chats. I'm using react-native-gifted-chat, which is heavily tied to the keyboard. Writing a patch to implement a custom keyboard feels risky due to how deeply the library relies on keyboard behavior.

So I wanted to ask — is there any alternative way to solve this problem, rather than replacing react-native-gifted-chat?

**Comment by gbumps** [comment]
@Kirillsocivka this problem will be fixed with an upcoming version `1.0.8`, reference: #85 for more info

**Comment by yogeshpoonia-ds** [comment]
```javascript
if (Platform.OS === 'ios') {
      ScreenGuardModule.register({
        backgroundColor: '#0F9D58',
        timeAfterResume: 100,
      });
    } else {
      ScreenGuardModule.registerWithoutEffect();
    }
```

worked for me

**Comment by gbumps** [comment]
This has been fixed and released at v2.0.0, please update and rebuilt.

Thanks!
